### ISSUE TITLE
FEATURE: separate config file for enable/disable of individual checks

### ISSUE THREAD

**Issue by astrostl** [SummitRoute/osxlockdown#13]
For those of us that like to keep in sync with the mothership, setting `enabled` to true or false within the `commands.json` (soon to be `commands.toml`?) file will require a local merge every time a pull is executed. If this were done in a separate file, one could keep that independent and pull at will.

**Comment by 0xdabbad00** [comment]
Agreed. I need to think of a smart way to keep track of versions of osxlockdown that takes into account the rules, allow people to potentially have their own additional rules, and allow them to enable and disable rules and maintain those preferences across new releases.

**Comment by astrostl** [comment]
I've always been fond of [*BSD's `.local` overrides](http://www.openbsd.org/cgi-bin/man.cgi/OpenBSD-current/man8/rc.conf.8?query=rc%2econf&sec=8): one file for stock settings, and a .local version which overrides those settings. Here, for example, a `commands.json` and a `commands.json.local` would be a solid model for my tastes. Internally, the app could load the defaults into a mutable data structure, then the override into the same.
